.github/workflows/helper/configureV8.js

```diff
@@ -0,0 +1,60 @@
+fs = require('fs');
+
+const buildGradlePath = 'app/android/app/build.gradle';
+const mainApplicationPath =
+  'app/android/app/src/main/java/com/app/MainApplication.java';
+
+fs.readFile(buildGradlePath, 'utf8', function (err, data) {
+  if (err) {
+    return console.log(err);
+  }
+  data = data.replace('enableHermes: true,', 'enableHermes: false,');
+
+  fs.writeFile(buildGradlePath, data, function (err) {
+    if (err) {
+      return console.log(err);
+    }
+  });
+});
```

Comment:
Can we wrap this in a separate function `patch(path, find, replace)`? This way we can eliminate code duplication. Then we can remove const strings and pass code snippets as inline arguments.